composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "cloudevents/sdk-php",
+    "description": "CloudEvents SDK",
+    "type": "library",
+    "license": "Apache-2.0",
+    "require": {
+        "php": ">=7.4"
```

Comment:
I'm torn between the nice features of 7.4 and supporting 7.3 to hopefully get greater adoption (though it's only getting security fixes now). Thoughts on this @jlaswell?

Reply:
Same here — the value of typed properties pushed me to say with 7.4 here. I think 7.4 the best option to help speed up the validation development at this point, but I'm open to a shift here.

Either way, I think it will also be important capture the support schedule post-initial release via the README.